src/users/routes.js

```diff
@@ -12,6 +12,7 @@ userRouter.post(
 userRouter.post(
 	"/users/register",
 	middleware.hashPassword,
+	middleware.validateUser,
```

Comment:
Eventually, you may want to add this before the hash password middleware. If you add more validation functions. For now, it's fine.